tools/submission/submission-checker.py

```diff
@@ -687,7 +687,7 @@ def get_mlperf_model(self, model):
             model = "ssd-small"
         elif "mobilenet" in model:
             model = "mobilenet"
-        elif "efficientnet" in model:
+        elif "efficientnet" in model or "resnet50" in model:
```

Comment:
I'd have this case should have been handled by code on lines 679-682 (at least, according to the comment):
```
        # simple mapping, ie resnet50->resnet ?
        mlperf_model = self.base["model_mapping"].get(model)
        if mlperf_model:
            return mlperf_model
```

Reply:
To my understanding, the goal of this PR is to be able to submit in the open division like it is done [here](https://github.com/mlcommons/inference_results_v1.1/tree/main/open/Krai/results/firefly-tflite-v2.5.0-ruy).

Where you can have multiple submissions for the same system with the model name varying slightly. Right now you can do this if the model name contains efficientnet. But now that resnet50 is being used, it would be useful if names like `resnet50-1`, `resnet50-2`... have the same behavior